### DISCUSSION TITLE
Chapter 5 observations

### DISCUSSION THREAD

**Discussion by Salizer** [aerickt/lapwing-for-beginners#2] [General]
Not yet done with the chapter, but found 2 oddities.

[Basic phonetics—single syllable words](https://lapwing.aerick.ca/Chapter-05.html#basic-phoneticssingle-syllable-words)

-     purr PUR
-     rough RUF

_Those last two words may have been a little bit tricky as their spellings contain multiple vowel letters._

Is it not only the last word that contains it? 

[Combining single left hand keys](https://lapwing.aerick.ca/Chapter-05.html#combining-single-left-hand-keys)
The practise material contains "l" like "plug" which is not explained yet.

By the way, I love what you have made :)

**Comment by aerickt** [comment]
Thanks, should be fixed now.
